src/transformer/mockDefiner/factoryUniqueName.ts

```diff
@@ -4,6 +4,8 @@ import { KeyCounter } from './keyCounter';
 export type PossibleDeclaration = ts.InterfaceDeclaration | ts.ClassDeclaration | ts.TypeAliasDeclaration;
 
 export class FactoryUniqueName {
+    private static AnonymousIdentifierText: string = '*';
```

Comment:
Can we put these values with the other one? Do you think it's a good idea to keep all these "reduced" identifier names in one place?

Reply:
done!